met-web/public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
     <head>
         <meta charset="utf-8" />
-        <title>Modern Engagement Tool</title>
+        <title>EPIC.engage</title>
```

Comment:
we might need to use React Helmet and make this title dynamic according to tenant. Could you check with Saravan if it's wise